apps/bulk_data_migration/entry_images_v2/migrate.py

```diff
@@ -0,0 +1,85 @@
+from django.db.models import Q
+from django.conf import settings
+
+from utils.common import parse_number
+
+from lead.models import LeadPreviewImage
+from entry.models import Entry
+from gallery.models import File
+
+"""
+python3 manage.py bulk_migrate entry_images_v2
+"""
+
+FILE_API_PREFIX = '{protocol}://{domain}{url}'.format(
+    protocol=settings.HTTP_PROTOCOL,
+    domain=settings.DJANGO_API_HOST,
+    url='/file/',
+)
+S3_URL_PREFIX = f'https://{settings.AWS_STORAGE_BUCKET_NAME_MEDIA}.s3.amazonaws.com/{settings.MEDIAFILES_LOCATION}/'
+
+
+def _get_file_from_file_url(entry, string):
+    fileid = parse_number(string.rstrip('/').split('/')[-1])
+    return fileid and File.objects.filter(id=fileid).first()
+    try:
+        fileid = parse_number(string.rstrip('/').split('/')[-1])
+        return fileid and File.objects.filter(id=fileid).first()
+    except IndexError:
+        pass
```

Comment:
Maybe, we only need `try...except` block here.

Reply:
Done